src/cinema/domainmodel/Statistic.java

```diff
@@ -0,0 +1,34 @@
+package cinema.domainmodel;
+
+public class Statistic {
+
+    private int currentIncome = 0;
+
+    private int numberOfAvailableSeats = 81;
```

Comment:
Shouldn't be calculated here, it should be passed as parameter for more agility of the solution.
https://en.wikipedia.org/wiki/Magic_number_(programming)

Reply:
fixed